docs/execute/IPv6.md

```diff
@@ -0,0 +1 @@
+Some scenarios requires the docker daemon to supports IPv6. It should be ok on Linux and Mac OS, but has not be tested on Windows.
```

Comment:
is this file referenced in somewhere?

Reply:
no, nowhere, what do you have in mind ?

Reply:
If not referenced from another page, this document does not explain much.
Or somewhere we explain the new scenario (docs/scenarios/ipv6.md).
Or from docs/execute/run.md, how to run scenarios over ipv6 and we reference this ipv6.md

Reply:
addressed in [3aae93a](https://github.com/DataDog/system-tests/pull/3624/commits/3aae93ab6692861a39a315dea348460a3119bb2b)